PowerEditor/src/WinControls/shortcut/shortcut.h

```diff
@@ -174,6 +174,14 @@ class Shortcut  : public StaticDialog {
 
 	void setName(const TCHAR * name);
 
+	void clear(){
+		_keyCombo._isCtrl = false;
+		_keyCombo._isAlt = false;
+		_keyCombo._isShift = false;
+		_keyCombo._key = 0;
+		return;
+	}

```

Comment:
It is probably worth changing the default constructor to call this method instead of duplicating the code.